src/azul/service/hca_response_v5.py

```diff
@@ -143,18 +164,26 @@ def for_bucket(cls, bucket: JSON):
         return bucket['key']
 
 
-class HitEntry(JsonObject):
-    pass
+class HitEntry(AzulJsonObject):
+
+    # By calling the JsonObject constructor with a single positional argument
+    # dictionary instead of keyword arguments we avoid a code path in jsonobject
+    # that makes a deep copy of the object.
```

Comment:
```suggestion
    # By passing a dictionary as the the sole positional argument instead of one keyword argument per dictionary entry we avoid a code path in jsonobject
    # that makes a deep copy of the object.
```

Reply:
This comment should be above the super call not above the method.